descqa/shear_test.py

```diff
@@ -0,0 +1,233 @@
+
+#from __future__ import unicode_literals, absolute_import, division
+import os
+import numpy as np
+from .base import BaseValidationTest, TestResult
```

Comment:
reorder all imports --- (1) built-in modules, (2) installed modules, (3) this module (those start with `from .`)